en/docs/learn/fraud-detection.md

```diff
@@ -0,0 +1,422 @@
+    learning techniques for implementing these strategies. Most of them provide APIs or libraries for any enterprise to
+    easily integrate with them.
+
+
+## Integrating with WSO2
+
+Within Identity Server, we have provided extensive capabilities to easily integrate with any third-party platform
+that provides any of the above services. There are a number of different ways to do the integration. The diagram
+given below depicts the architecture of the WSO2 authentication framework.
+
+![authentication-framework](../assets/img/learn/authentication-framework.png)
+
+In the authentication flow shown above, the following extension points are available for integrating with an external
+ system.
+
+### Publishing Authentication Events
+
+#### Event Publishers
+
+In Identity Server **events** are used to analyze data effectively. Event is a unit of data. An **event stream** is a
+sequence of a particular type of event. The type of events can be defined as an **event stream definition**.
+Following is a sample event stream for authentication data.
+
+```json
+    {
+      "name": "org.wso2.is.analytics.stream.OverallAuthentication",
+      "version": "1.0.0",
+      "nickName": "",
+      "description": "",
+      "metaData": [
+        {
+          "name": "tenantId",
+          "type": "INT"
+        }
+      ],
+      "payloadData": [
+        {
+          "name": "contextId",
+          "type": "STRING"
+        },
+        {
+          "name": "eventId",
+          "type": "STRING"
+        },
+        {
+          "name": "eventType",
+          "type": "STRING"
+        },
+        {
+          "name": "authenticationSuccess",
+          "type": "BOOL"
+        },
+        {
+          "name": "username",
+          "type": "STRING"
+        }
+      ]
+    }
+```
+
+The event streams that are supported by default in Identity Server, are stored in the filesystem as deployable
+artifacts in the `<IS_HOME>/repository/deployment/server/eventstreams` directory.
+
+Identity Server has the ability to publish these events to analytics servers in multiple formats such as **HTTP**,
+**log**, **wso2event** etc. Furthermore, it has a default set of data publishers to publish different kinds of data such
+ as authentication data, session data, oauth token related data, user and role data. You can refer
+`<IS_HOME/repository/deployment/server/event-publishers` to understand the different types of data publishers we have.
+By default these publishers are configured to publish event streams to **WSO2 IS Analytics Server**. More information
+ on how you can configure IS Analytics Server with IS can be found [here](https://is.docs.wso2.com/en/5.10
+ .0/learn/prerequisites-to-publish-statistics).
+
+However if you want to configure thesedata publishers with external **Security information and event management (SIEM)**
+ softwares such as **Splunk**, **IBM QRadar** and **LogRhythm**, you can easily modify the configurations in these XML
```

Comment:
....such as Splunk, IBM QRadaer, and LogRhythm, modify the configurations in these XML-based...